app/test/frontend/golden/pkg_show_version_page.html

```diff
@@ -66,7 +66,7 @@ <h1 class="_visuallyhidden">Dart pub</h1>
               </div>
             </div>
           </nav>
-          <div></div>
+          <div>&amp;nbsp;</div>
```

Comment:
this seems weird

Reply:
Yes, probably because we've upgraded some package.. this one I can't even explain.

Reply:
Ah, https://pub.dev/packages/xml#360

Minor change in `package:xml`, fixed it...